gitops/sre/.tekton/kcp-registration.yaml

```diff
@@ -50,6 +50,8 @@ spec:
                   value: $(workspaces.source.path)
                 - name: KCP_ORG
                   value: "pipelines-service"
+                - name: KCP_WORKSPACE
```

Comment:
Don't we have to add KCP_TAG here too?

Reply:
No we don't have because of [this](https://github.com/openshift-pipelines/pipelines-service/blob/main/images/kcp-registrar/Dockerfile#L5-L6)

Reply:
This is actually wrong. The Tag is only set in the builder image.